lib/ronin/recon/builtin/web/dir_enum.rb

```diff
@@ -99,9 +99,12 @@ def process(website)
                     headers  = response.headers.to_h
 
                     if VALID_STATUS_CODES.include?(status)
+                      body = http.get(path).read
```

Comment:
So `async-http` has this weird design where you must call `.close` on any `response` object it returns, otherwise it creates a resource leak. Will need to do something like:

```ruby
response = http.get(path)
body     = begin
             response.read
           ensure
             response.close
           end
```